src/components/backButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {TouchableOpacity, StyleSheet} from 'react-native';
+import CloseBlack from '@assets/Cross.svg';
+import BackArrow from '@assets/Back_Arrow.svg';
+
+interface IBackButtonProps {
+  onClick: () => void;
+  isCloseButton?: boolean; // true for cross button else normal back button
+}
+
+const bbStyle = StyleSheet.create({
+  container: {
+    padding: 10,
+    paddingLeft: 0,
+    alignItems: 'center',
+  },
+});
+
+export const BackButton: React.FC<IBackButtonProps> = ({
+  onClick,
+  isCloseButton,
+}) => {
+  const BACK_IMAGE = isCloseButton ? CloseBlack : BackArrow;
+  return (
+    <TouchableOpacity
+      onPress={onClick}
+      activeOpacity={1}
+      style={bbStyle.container}>
+      <BACK_IMAGE width={35} height={35} />
```

Comment:
Max of buttons will come with same sizes, declare this in global config and use {...} spread operator to assign it has props.